lib/events/pgevents/pgevents_test.go

```diff
@@ -38,52 +45,127 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+// TELEPORT_TEST_PGEVENTS_URL is connection URL similar to the one used by
+// "audit_events_uri" (in teleport.yaml).
+// For example: "postgresql://teleport@localhost:5432/teleport_test1?sslcert=/path/to/cert.pem&sslkey=/path/to/key.pem&sslrootcert=/path/to/ca.pem&sslmode=verify-full&pool_max_conns=20"
 const urlEnvVar = "TELEPORT_TEST_PGEVENTS_URL"
 
 func TestPostgresEvents(t *testing.T) {
-	s, ok := os.LookupEnv(urlEnvVar)
-	if !ok {
-		t.Skipf("Missing %v environment variable.", urlEnvVar)
-	}
-
-	u, err := url.Parse(s)
-	require.NoError(t, err)
-
-	var cfg Config
-	require.NoError(t, cfg.SetFromURL(u))
-
-	ctx, cancel := context.WithCancel(context.Background())
-	t.Cleanup(cancel)
-
-	log, err := New(ctx, cfg)
-	require.NoError(t, err)
-	t.Cleanup(func() { require.NoError(t, log.Close()) })
+	// Don't t.Parallel(), relies on urlEnvVar.
```

Comment:
The fact that it depends on reading an envvar doesn't mean that it's not parallelizable; the fact that it uses the same audit log as other tests (and this one in particular will outright clear the data repeatedly) does.

```suggestion
	// Don't t.Parallel(), uses the audit log at urlEnvVar like other tests.
	t.Setenv("_TELEPORT_TEST_NO_PARALLEL", "1")
```